cmd/gpu_plugin/README.md

```diff
@@ -32,30 +32,29 @@ device-plugin: ListAndWatch start
 ListAndWatch: send devices &ListAndWatchResponse{Devices:[&Device{ID:card0,Health:Healthy,}],}
 ```
 
-### Check if GPU device plugin is registered on master
+### Verify GPU device plugin is registered on master:
 ```
 $ kubectl describe node <node name> | grep intel.com/gpu
  intel.com/gpu:  1
  intel.com/gpu:  1
 ```
 
-### Test GPU device plugin
+### Test GPU device plugin:
 
 1. Build a Docker image with beignet unit tests:
-
    ```
    $ cd demo
    $ ./build-image.sh ubuntu-demo-opencl
    ```
 
-   This command will produce a Docker image named `ubuntu-demo-opencl`.
+      This command produces a Docker image named `ubuntu-demo-opencl`.
 
 2. Create a pod running unit tests off the local Docker image:
    ```
    $ kubectl apply -f demo/intelgpu_job.yaml
    ```
 
-3. Observe the pod's logs:
+3. Review the pod's logs:
```

Comment:
Add details about what they're looking for in the logs or is it obvious pass/fail data?

Reply:
It's the latter.